src/tag_serial_driver.cpp

```diff
@@ -92,29 +92,42 @@ int serial_setup(const char * device)
   return fd;
 }
 
-void receive_ver_req(const std_msgs::msg::Int32::ConstSharedPtr msg)
+void receive_ver_req([[maybe_unused]] const std_msgs::msg::Int32::ConstSharedPtr msg)
 {
   char ver_req[] = "$TSC,VER*29\x0d\x0a";
-  write(fd, ver_req, sizeof(ver_req));
-  RCLCPP_INFO(rclcpp::get_logger("tag_serial_driver"), "Send Version Request:%s", ver_req);
+  int ver_req_data = write(fd, ver_req, sizeof(ver_req));
+  if (ver_req_data >= 0) {
+    RCLCPP_INFO(rclcpp::get_logger("tag_serial_driver"), "Send Version Request:%s", ver_req);
```

Comment:
[Nits] It's better to use the same style.

![image](https://user-images.githubusercontent.com/31987104/154049330-190e014a-ff09-41f3-b414-cdef334cdbe1.png)

```suggestion
    RCLCPP_INFO(rclcpp::get_logger("tag_serial_driver"), "Send Version Request: %s", ver_req);
```

Reply:
fixed in https://github.com/tier4/tamagawa_imu_driver/pull/5/commits/c97a171574042a6c5939f7dbda37da5c7e074f43